.gitignore

```diff
@@ -135,3 +135,5 @@ dist
 docs/
 apidocs/
 devdocs/
+
+.idea/
```

Comment:
Please add this to your .gitignore locally instead of for everyone, you can do so following the advice here:

https://dev.to/usmanzaheer1995/git-ignore-files-locally-327i